src/Spice86.Core/Emulator/Gdb/GdbCustomCommandsHandler.cs

```diff
@@ -380,7 +380,8 @@ public class GdbCustomCommandsHandler {
                     return _gdbIo.GenerateMessageToDisplayResponse($"Buffer already exists: {existing}");
                 }
 
-                gui?.AddBuffer(_machine.VgaCard, address, scale, resolution[0], resolution[1]);
+                // Todo: Fix this?
+                // gui?.AddBuffer(_machine.VgaIoPortHandler, address, scale, resolution[0], resolution[1]);
```

Comment:
I think you can remove the feature altogether, it was never very useful

Reply:
It would simplify the UI code.

Reply:
Phew, that's good to hear. Although it should not be too hard to re-implement something like this after everything has settled.